test/integration/render-error-on-render-error/with-get-initial-props/pages/_error.js

```diff
@@ -0,0 +1,11 @@
+const Error = ({ message }) => {
```

Comment:
I'm not super happy with the test suite name `render-error-on-render-error`, open to feedback on a better name 😄